### ISSUE TITLE
Cannot find module 'har-validator'

### ISSUE THREAD

**Issue by erwanriou** [cypress-io/cypress#19097]
### Current behavior

**AS** a user or a action builder 
**WHEN** installing cypress version **8.7.0** 
**AND** within a docker environment

i have the following logs:

```sh
npm ERR! code 1
npm ERR! path /app/node_modules/cypress
npm ERR! command failed
npm ERR! command sh -c node index.js --exec install
npm ERR! node:internal/modules/cjs/loader:903
npm ERR!   throw err;
npm ERR!   ^
npm ERR! 
npm ERR! Error: Cannot find module 'har-validator'
npm ERR! Require stack:
npm ERR! - /app/node_modules/@cypress/request/lib/har.js
npm ERR! - /app/node_modules/@cypress/request/request.js
npm ERR! - /app/node_modules/@cypress/request/index.js
npm ERR! - /app/node_modules/cypress/lib/tasks/download.js
npm ERR! - /app/node_modules/cypress/lib/tasks/install.js
npm ERR! - /app/node_modules/cypress/index.js
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:900:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:745:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:972:19)
npm ERR!     at require (node:internal/modules/cjs/helpers:88:18)
npm ERR!     at Object.<anonymous> (/app/node_modules/@cypress/request/lib/har.js:5:16)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1083:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:948:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:789:14)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:972:19)
npm ERR!     at require (node:internal/modules/cjs/helpers:88:18)
npm ERR!     at Object.<anonymous> (/app/node_modules/@cypress/request/request.js:23:11)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1083:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:948:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:789:14) ***
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     '/app/node_modules/@cypress/request/lib/har.js',
npm ERR!     '/app/node_modules/@cypress/request/request.js',
npm ERR!     '/app/node_modules/@cypress/request/index.js',
npm ERR!     '/app/node_modules/cypress/lib/tasks/download.js',
npm ERR!     '/app/node_modules/cypress/lib/tasks/install.js',
npm ERR!     '/app/node_modules/cypress/index.js'
npm ERR!   ]
npm ERR! ***
```

### Desired behavior

**AS** a user or a action builder 
**WHEN** trying to installing cypress version **8.7.0** 
**AND** within a docker environment

The installation should be successfull

### Test code to reproduce

npm i --save cypress@8.7.0

### Cypress Version

8.7.0

### Other

Tested within a docker environment with nodejs 15.0.1 alpine

**Comment by carlosq-ordering** [comment]
Same here 😢

**Comment by mrsGehred** [comment]
Same here for 9.1.0, 9.0.0 :-(

**Comment by CarlosOrdering** [comment]
same here 🥺

**Comment by cluk3** [comment]
Same here, v6.4.0 both on docker and on my mac

**Comment by erwanriou** [comment]
sound like the last commit done 1 hour ago is failing hard. We probably can expect a revert or a fix soon as it's global.

**Comment by emilyrohrbough** [comment]
@erwanriou  Would you be willing to open a PR to fix this? We'd truly appreciate it!

**Comment by CannyBangbang** [comment]
Also hitting with 8.7.0 both with docker and mac

**Comment by erwanriou** [comment]
yes @grxy was faster than me and took care of it :)

**Comment by grxy** [comment]
@erwanriou Didn't mean to step on toes. Thanks for the ✅

**Comment by thepuzzlemaster** [comment]
Is there a way to install cypress in the meantime before this fix gets merged in? I would have thought that just installing an older version from before the removal would have worked, but that doesn't seem to be the case.

**Comment by arthurgeek** [comment]
@thepuzzlemaster you have to pin `"@cypress/request"` version, not `cypress`.

add

```
"@cypress/request": "2.88.7",
```

to your package.json and it should work.

**Comment by arifamd00** [comment]
I am facing same issue.. I have urgent project to work upon. Any solution?

**Comment by guilhermetelles** [comment]
@arifamd00 look at the comment above, it's a temporary fix


> @thepuzzlemaster you have to pin `"@cypress/request"` version, not `cypress`.
> 
> add
> 
> ```
> "@cypress/request": "2.88.7",
> ```
> 
> to your package.json and it should work.

**Comment by emilyrohrbough** [comment]
This was resolved in [cypress-io/request#16 released ](https://github.com/cypress-io/request/pull/16) and released as [@cypress/request@v2.88.10](https://github.com/cypress-io/request/releases/tag/v2.88.10).
